CHANGELOG.md

```diff
@@ -9,6 +9,10 @@
 - Added `memoToNeuronSubaccount` and `memoToNeuronAccountIdentifier`.
 - Support new neuron field `voting_power_refreshed_timestamp_seconds`.
 
+## Build
+
+- Update the dev dependencies of the `ic-js` monorepo.
```

Comment:
Why do you call it the monorepo?

Reply:
I wanted to highlight the fact that ic-js itself is not a library per sé. I can use the term "workspaces" if you rather like?

Reply:
Updated to "workspace" to move on and merge the PR.

Reply:
Yes, "workspace" makes sense.
To me a monorepo encompasses much more. The fact that nns-dapp and ic-js aren't even in the same repo makes it definitely not a monorepo. But I don't know if my understanding of "monorepo" is "correct".

Reply:
It's probably a fine grain here, probably depends if we consider our libs as projects or not. Anyway, for sure I'm fine with the change, thanks for the feedback.